css-basics_social-card/css/styles.css

```diff
@@ -14,8 +14,8 @@ body {
 }
 
 .content {
-  width: 50%;
-  height: auto;
+  width: 95%;
+  height: 50%;
```

Comment:
Similar comment for the line above, the height will be defined by the content itself.